### DISCUSSION TITLE
Scan stopped importing new video files

### DISCUSSION THREAD

**Discussion by DanSixtyNine** [stashapp/stash#5515] [Q&A] [answered]
Stash just stopped importing new video files as scenes a few weeks ago when scanning. Old files are still displayed correctly, and new pictures are also correctly imported.

**Comment by Maista6969** [comment, marked as answer]
Try deleting the old ffmpeg/ffprobe executables in the Stash folder (by default called `.stash` in your home directory) and having Stash download new versions through `Settings > System > Application Paths`
![bilde](https://github.com/user-attachments/assets/8e353dac-15f9-477e-8c9b-814545884231)

**Comment by DanSixtyNine** [comment]
That worked, thank you!
